programs/cli_wallet/main.cpp

```diff
@@ -72,6 +72,9 @@ int main( int argc, char** argv )
 {
    try {
 
+      const char* regex_secret_name = "regex-secret";
+      const string regex_secret = "^(unlock|set_password)\\s.*";
```

Comment:
Must allow ``\s*`` at the start of the line.

Reply:
Done ! Thanks !